### PR TITLE
Fix/build

### PR DESCRIPTION
**Pull request by szuecs**
debug with git status

### CODE REVIEW

**Comment by szuecs** [comment]
:+1:

**Comment by aryszka** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Comment by szuecs** [comment]
The fix was to run locally:
`GO111MODULE=on go mod tidy`
and add the changed go.mod and go.sum files

File: delivery.yaml [hunk 1]
```diff
@@ -14,10 +14,14 @@ pipeline:
       fi
       export IMAGE
       make deps shortcheck
-      cd packaging && make docker-build docker-push
+      git status
+      git diff
```
Comment (aryszka) [inline delivery.yaml]: shall we keep the diff in the build?

**Comment by aryszka** [comment]
:+1:

File: delivery.yaml (hunk 1 above)
Comment (szuecs) [reply delivery.yaml]: let's merge now and see if this really fixed it for the merge to master case :)

**Pull request merged**